gradio/routes.py

```diff
@@ -379,61 +375,60 @@ async def predict(
 
             # If this fn_index cancels jobs, then the only input we need is the
             # current session hash
-            if app.blocks.dependencies[body.fn_index]["cancels"]:
+            if app.get_blocks().dependencies[body.fn_index]["cancels"]:
                 body.data = [body.session_hash]
             if body.request:
                 if body.batched:
-                    request = [Request(**req) for req in body.request]
+                    request_ = [Request(**req) for req in body.request]
```

Comment:
I've always seen the underscore come first when avoiding another namespace, e.g. `_id` instead of `id`.

Reply:
Sure thanks